pymc/sampling.py

```diff
@@ -1835,36 +1847,34 @@ def sample_posterior_predictive(
         idata_kwargs = {}
     else:
         idata_kwargs = idata_kwargs.copy()
+    if sample_dims is None:
+        sample_dims = ["chain", "draw"]
```

Comment:
If you use a tuple you can use it directly as the default value in the signature